<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Images</class>
 <widget class="QWidget" name="Images">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#images{
	background:#333;
}
</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>761</width>
     <height>501</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#groupBox{
	background:#333;
}

*{
	font-family:Century gothic;

}

QPushButton{
	border-radius:15px;
	background:blue;
	color:white;
}

QPushButton:hover{
	border-radius:15px;
	background:#49ebff;
	color:#333;
}
QLabel{
	border:1px solid white;
	border-radius:10px;
}


</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="img1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>161</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="p1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>290</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Image 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="p2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>290</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Image 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="p3">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>290</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Image 3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="p4">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>290</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Image 4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="confirm">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>390</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Confirmer</string>
    </property>
   </widget>
   <widget class="QLabel" name="img2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>100</y>
      <width>161</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="img3">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>100</y>
      <width>161</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="img4">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>100</y>
      <width>161</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>401</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
color:white;
font-size:20px;
font:bold;</string>
    </property>
    <property name="text">
     <string>Ajouter les images d'un employe</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
